EasyVCR.Tests/ClientTest.cs

```diff
@@ -317,6 +318,44 @@ public async Task TestXmlCensoring()
             // have to manually check cassette for the censored string in the response body
         }
 
+        [TestMethod]
+        public async Task TextIgnoreElements()
```

Comment:
This test is doing two things, it checks if it fails without ignoring and then checks that it passes when differing things get ignored. These should really be split up into two separate tests, one testing each piece just mentioned individually.